server/utils/importStig.ts

```diff
@@ -2,54 +2,96 @@ import * as fs from "fs";
 import { readFileSync } from "fs";
 import path from "path";
 import { parseStringPromise } from "xml2js";
-import { UniqueConstraintError } from "sequelize";
-import { Stig, StigLibrary, StigLibraryInterface } from "../../db/models";
-import { hashFile } from "./hash";
+import { Transaction, UniqueConstraintError } from "sequelize";
+import { Stig, StigIdent, StigLibrary, StigReference, StigResponsibility } from "../../db/models";
```

Comment:
Suggest removing relative pathing on import

```suggestion
import { Stig, StigIdent, StigLibrary, StigReference, StigResponsibility } from "~/db/models";
```